### DISCUSSION TITLE
workflow with "on: push" triggers on release?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#25048] [Actions]
<p>I have a workflow setup which triggers when I push files to the path “wiki/**” to then update the wiki of the repository.</p>
<p>But I now created a release and all of a sudden did this action get triggered by this.</p>
<p>Do releases now count as pushes or is this an unintentional behaviour or perhaps a visual bug?</p>
<p>I don’t believe last option as it shows as a separate action in the Actions-tab with the same commit hash:</p>
<p><img src="https://user-images.githubusercontent.com/11576465/181088555-05c9824b-15c9-4c9c-a35e-22075133b3f8.png" alt="actions.png" data-base62-sha1="aw6uJ7L5eY0QbreTFHcSTcBXXSc" width="168" height="180"></p>
<p>The “on” part is set up as follows:</p>
<pre><code>on:
  push:
    paths:
      - 'wiki/**'
</code></pre>
<p>If that is actually an intended behaviour, could someone give me a solution to make this ignore releases?</p>

**Comment by peter-evans** [comment, marked as answer]
<p>A release triggers <code>on: push</code> because it pushes a tag. I think you can ignore pushes to tags and just have the workflow triggered on pushes to branches with this filter.</p>
<pre><code>on:
  push:
    branches:
      - '**'
</code></pre>

**Comment by Andre601** [comment]
<p>Thanks.</p>
<p>I found a similar solution.</p>
<p>GitHub offers “tags-ignore” to ignore specific tags. So by using the snipped below was I able to make the action ignore all releases.</p>
<pre><code>on:
  push:
    paths:
      - 'wiki/**'
    tags-ignore:
      - '**' # Just ignore all tags.
</code></pre>
